core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -944,6 +944,7 @@ protected Jenkins(File root, ServletContext context, PluginManager pluginManager
 
 
             if(KILL_AFTER_LOAD)
+                // TODO cleanUp?
```

Comment:
I doubt it is really required since running with this flag on a production instance is a quite exotic idea, no?